src/mon/MonCommands.h

```diff
@@ -155,6 +155,7 @@ COMMAND("pg ls-by-osd " \
 	"list pg on osd [osd]", "pg", "r", "cli,rest")
 COMMAND("pg ls " \
         "name=pool,type=CephInt,req=false " \
+        "name=osd,type=CephInt,req=false " \
```

Comment:
i don't really think we should expose the implementation details in the help message.